change/office-ui-fabric-react-2020-09-01-11-04-49-master.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
+  "comment": "Expose onRenderColorItem and override",
```

This goes in the release notes, so it's good to include which component is being modified.
```suggestion
  "comment": "SwatchColorPicker: Expose onRenderColorItem and override",
```

---

Thanks for the feedback. Updated the comment as requested